pkg/resource/deploy/snapshot.go

```diff
@@ -175,3 +178,45 @@ func (snap *Snapshot) VerifyIntegrity() error {
 
 	return nil
 }
+
+// Performs glob style expansion on urns that contain '*'. Each urn can be
+// expanded into 1-n actual urns, depending on what underlying resources exist
```

Comment:
```suggestion
// expanded into 0-n actual urns, depending on what underlying resources exist
```
There might be no matches